standards.py

```diff
@@ -7,6 +7,12 @@
 import argparse
 import datetime
 
+"""
+For Mac users run:
+>brew install dotnet
+To install dotnet-format, run:
+>dotnet tool install --global dotnet-format 
+"""
```

Comment:
this all makes sense but I'm planning to write-down docs in `CODING_STANDARDS.md` soon and I believe this would be a section in there.

Reply:
Could we merge this now and you integrate this in your own doc when it's available?
Else I can just close this PR and let you integrate these mac instructions yourself when you're ready.

Reply:
I think I'll get something ready and ping you over there either this week or next week :)